Prepare release v1.0.3 (#829)
Deltares/hydromt_wflow: docs/user_guide/5_setup_methods/setup_ponding.rst

.. currentmodule:: hydromt_wflow

Setup Ponding
=============

.. note::

   The methods ``setup_ponding_from_map`` and ``setup_ponding_from_threshold``
   will be added in a future release when wflow.jl supports it. The
   documentation for these methods is already included here in anticipation
   of that support.

Description
-----------
Many nature-based solutions (NBS) for water management share a common hydrological mechanism:
the **temporary storage (ponding) of water on the land surface**, followed by **enhanced infiltration
and groundwater recharge**. Although such measures differ in form and scale, their effect is to
slow overland flow, increase residence time, and promote re-infiltration into the soil.

> To better represent these processes, new functionality has been added to the wflow model.

Reviewer: I guess this is not so "new" anymore after a few releases right? I think we can leave words like "new" to the changelog, as it has a date in the header giving some context to when something was "new".

```suggestion
The wflow model has functionality to represent these processes.
```

First, **re-infiltration of overland flow** is now allowed, enabling water that would previously
be routed laterally downstream to infiltrate back into the soil. Second, a **ponding threshold**
has been implemented: below a specified water depth, overland flow does not move laterally
but ponds on the land surface, increasing the potential for infiltration and recharge. When
this threshold is exceeded, excess water resumes lateral flow.

This ponding concept provides a flexible, process-based representation of a wide range of
NBS measures that involve surface water retention and infiltration, including:

- rainwater harvesting
- terracing
- small ponds in forested hillslope
- plastic-lined water harvesting ponds in cropland area
- hedgerows / stone lines / springshed revival through trenches and check dams in croplands areas
- soil bunds / demi-lunes

> As a start, two methods have been implemented to set up ponding level in hydromt wflow:

Reviewer: ```suggestion
As a start, two methods were implemented to set up ponding level in hydromt wflow:
```
